def teste():
    x = 8
    print(f'Na função teste, n vale {n}')
    print(f'Na funçao teste, x vale {x}')
 
 
 #PROGAMA PRINCIAPL
n = 2
print(f'No progama princiapl, n vale {n}')
teste()
print(f'No progama princiapl, x vale {x}')
#VAI DA ERROR
